### ISSUE TITLE
Velocity support 

### ISSUE THREAD

**Issue by zentixua** [voidpointer0x00/VoidWhitelist#23]
Hello. I have a fabric server and I would like that whitelist was with this server is common. But this will not work on Fabric and I will have to add everyone again to vanilla whitelist. But our servers are behind Velocity and I think I'm not the only one:)

**Comment by voidpointer0x00** [comment]
Yeah, other platforms support is on my todo list, I'll make that happen sooner or later.

Right now the codebase needs some refactoring, right after that I'll try and make the project more modular, so it will be easier to port.
